.github/workflows/ci.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
```

Comment:
does this actually not run on 16 or are we just removing it because 16 ist post-LTS

Reply:
fetch was introduced in 16.8(?) so it's easier for use to say "Nano 11 is for Node 18 and up"